components/nav/Header.vue

```diff
@@ -264,10 +266,12 @@ export default {
 
         <button
           v-if="showKubeShell"
-          v-tooltip="t('nav.shell')"
+          v-tooltip="t('nav.shell', {key: shellShortcut})"
+          v-shortkey="{windows: ['ctrl', '\''], mac: ['meta', '\'']}"
```

Comment:
On vscode, its ctrl+` on mac as well

Reply:
As per last conversation with @nwmac , we will leave the meta key in the mac env (to be consistent with the current header shortcut notation for the Resource Search) and just replace ' with `.

Reply:
<img width="1545" alt="Screenshot 2022-01-24 at 09 51 53" src="https://user-images.githubusercontent.com/97888974/150760472-f269bc5c-c572-4346-879c-5e56881b2270.png">

Reply:
As per limitations on Mac OS + Chrome, it's back to CTRL+`